addons/web/static/src/legacy/js/fields/basic_fields.js

```diff
@@ -695,7 +700,7 @@ var FieldChar = InputField.extend(TranslatableFieldMixin, DynamicPlaceholderFiel
             ev.preventDefault();
             const baseModel = this.recordData && this.recordData.mailing_model_real ? this.recordData.mailing_model_real : undefined;
             if (baseModel) {
-                await this.openDynamicPlaceholder(baseModel);
+                await this._openNewModelSelector(baseModel, []);
```

Comment:
It is not clear to me why you need to pass `[]` as an argument here ?

Reply:
it's not necessary indeed